Relazione/RelazioneTecWeb.tex

```diff
@@ -76,6 +76,11 @@
 	\section{Implementazione}
 		\subsection{Linguaggi}
 			\subsubsection{XHTML 1.0 Strict e HTML5}
+			Come già detto, per la realizzazione del sito si è optato per avere sia pagine in XHTML 1.0 Strict, che in HTML5. L'uso di XHTML 1.0 garantisce una migliore retrocompatibilità con browsers più datati ed una migliore aderenza del codice agli standard di correttezza, ma HTML5 permette di sfruttare alcune funzionalità aggiuntive che abbiamo sfruttato ad esempio in vari form per inserire \textit{placeholder} o per rendere dei campi \textit{required} in modo da chiarire e migliorare l'usabilità del sito.
+			\newline In generale, per garantire un codice corretto, sono state seguite le linee guida che sono state illustrate durante il corso di Tecnologie Web e quelle dello standard W3C. Il codice è stato poi validato utilizzando il tool di validazione W3C Validator.%
```

Comment:
Ho visto in un'altra relazione gli hanno scritto i punti più importanti e forse sarebbe carino metterglieli così gli facciamo vedere che sappiamo cosa bisogna fare.
![image](https://user-images.githubusercontent.com/56229661/70461830-d307d800-1ab9-11ea-8323-32abdab58fd2.png)

Reply:
Visto che anche per la sezione Obiettivi ho seguito quella relazione, ho cercato di mettere questa parte in modo più discorsivo per evitare che si assomiglino troppo.